site/content/integrate/apps/quickstart/_index.md

```diff
@@ -1,18 +1,67 @@
 ---
-title: Quick start
-description: Guides for quickly getting started building apps
+title: Apps - Developer setup
```

Comment:
I think the extra "Apps" looks redundant in the LHS. Would the reader not know they're in the Apps section?

Reply:
It's not redundant in the page title, which could benefit from the clarity. @mickmister makes a good point that if you open up a few tabs, it's hard to tell what is what in Chrome just by their names. I think this is something we would need to address across all of the docs, but I think it works okay here for now. 1/5

Reply:
Agreed. :)